plot/src/main/java/smile/swing/FontChooser.java

```diff
@@ -484,7 +483,7 @@ public TextFieldKeyHandlerForListSelectionUpDown(JList<String> list) {
 
         @Override
         public void keyPressed(KeyEvent e) {
-            int i = targetList.getSelectedIndex();
+            int i;
             switch (e.getKeyCode()) {
```

Comment:
This assignment for example is redundant and has no effect which reduces the readability.